pcdsdevices/dc_devices.py

```diff
@@ -0,0 +1,112 @@
+"""
+Standard classes for LCLS DC power devices.
+"""
+import logging
+from ophyd import EpicsSignal, EpicsSignalRO, Device
+from ophyd import FormattedComponent as FCpt
+
+from .interface import BaseInterface
+
+logger = logging.getLogger(__name__)
+
+
+class ICTChannel(Device):
+    """
+    Class to define a particular channel of the ICT
+
+    Parameters
+    ---------
+    prefix : ``str``
+        The PV base of the relevant ICT
+
+    channel : ``str``
+        The output channel on the ICT, e.g. '1A', '3B', etc
+
+    Usage
+    ---------
+    ICTChannel('TST:PWR:ICT:01', '2B' )
+    """
+    Ch_Current = FCpt(EpicsSignalRO, '{self.prefix}:Output{self._ch}:Current',
+                      kind='hinted')
+    Ch_Status = FCpt(EpicsSignal, '{self.prefix}:Output{self._ch}:GetState',
```

Comment:
Your functions use `ch_status`, not `Ch_Status`. I suggest changing all the names to lowercase to match the python conventions.

Reply:
oops I did that while playing around with typhon, meant to set those back.